src/cc_toolchain/cc_toolchain_config.bzl

```diff
@@ -192,7 +192,7 @@ def _make_common_features(ctx):
                 actions = [ACTION_NAMES.c_compile, ACTION_NAMES.cpp_compile],
                 flag_groups = [
                     flag_group(
-                        flags = ["-Wall", "-Wextra", "-Wvla", "-Wconversion"] +
+                        flags = ["-fdiagnostics-color=always", "-Wall", "-Wextra", "-Wvla", "-Wconversion"] +
```

Comment:
I chose to put it in the warnings group because it's the most related group I could find

Reply:
Can we add a new group instead? I don't think this is logically part of any of the existing groups.

Reply:
done